### PR TITLE
Feature/key hash api

### PR DESCRIPTION
**Pull request by jdiouf**
Added api to fetch experiences by metadata_key_hash

### CODE REVIEW

File: app/api/views.py [hunk 1]
```diff
@@ -62,6 +62,21 @@ def metadata_list(request):
                 }
 
                 return Response(errorMsg, status.HTTP_400_BAD_REQUEST)
+
+        # case a list of metadata key hashes sent as query parameter e.g a,b,c
```
Comment (mattdrayer) [inline app/api/views.py:66]: This comment seems slightly out of place -- it should be down on line 69

File: app/api/views.py [hunk 2]
```diff
@@ -62,6 +62,21 @@ def metadata_list(request):
...
+        if request.GET.get('metadata_key_hash'):
+            metadata_key_hash_param = request.GET.get('metadata_key_hash')
+            hashes = metadata_key_hash_param.split(',')
```
Comment (mattdrayer) [inline app/api/views.py:69]: Nit: maybe a better name like `metadata_hash_key_list`

File: app/api/views.py [hunk 3]
```diff
@@ -62,6 +62,21 @@ def metadata_list(request):
...
+            querySet = querySet.filter(metadata_key_hash__in=hashes)
+
+            if not querySet:
+                errorMsg = {
+                    "message": "Error; no metadata key hash found for: "
```
Comment (mattdrayer) [inline app/api/views.py:74]: This error might read oddly if there were multiple keys specified.

File: docker-compose.yml [hunk 4]
```diff
@@ -41,6 +41,7 @@ services:
       CELERY_RESULT_BACKEND: "${CELERY_RESULT_BACKEND}"
     volumes:
       - ./app:/opt/app/openlxp-xis
+      - ${CERT_VOLUME}:/etc/ssl/certs/ca-certificates.pem
```
Comment (mattdrayer) [inline docker-compose.yml:44]: Is this related to the PR or just an add-on?  If it's an add-on you might do it as a separate commit with its own message (can still be part of the PR).

**Review by mattdrayer — APPROVED** [review]
Just some style things.

File: docker-compose.yml (hunk 4 above)
Comment (jdiouf) [reply docker-compose.yml:44]: This is to solve an issue we've been having when deploying as well as locally, where the connection to AWS fails due to certificate issues. Will remove and commit separately.

**Review by KarenAJ — APPROVED** [review]
Approved!
